PriceKataCalc/Program.cs

```diff
@@ -15,17 +15,25 @@ class product{
         public String name;
         public int UPC;
         public double price;
+        public double discount;
```

Comment:
these properties shouldn't be public right ?

Reply:
i should've used getters and setters :(